src/app.ts

```diff
@@ -1,14 +1,22 @@
 import express, { Express } from 'express';
 
+import UserService from './services/users.service';
+import usersRouter from './routes/users.router';
+import UsersMiddleware from './middleware/users.middleware';
+import UsersController from './controllers/users.controller';
+import UserRepository from './repositories/userRepository';
 import type { DataBaseI } from './types/database';
 
-// TODO: pass DB to repository
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 function app (db: DataBaseI): Express {
-
   const application: Express = express();
 
+  const userRepository: UserRepository = new UserRepository(db);
```

Comment:
In TypeScript, there's no need to explicitly declare the type when the variable is being assigned an instance of that type. The type is inferred automatically